packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.base.tsx

```diff
@@ -24,8 +24,6 @@ export class ShimmeredDetailsListBase extends React.Component<IShimmeredDetailsL
   constructor(props: IShimmeredDetailsListProps) {
     super(props);
 
-    initializeComponentRef(this);
```

Comment:
@KevinTCoughlin removing this because we don’t need in here. `DetailsList` will call it instead.